### ISSUE TITLE
Machine Vision incorparation with yolact++ software and 3D pose calculation

### ISSUE THREAD

**Issue by minewarriorsSchool** [dbolya/yolact#606]
First of all, i am so amazed by how amazing this software is. It is insane how well it is built. 

Currently I am doing research on how I can use the real time detection of yolact++ and use this detection
to calculate a 3D pose with 2 machine vision camera's.

I have the yolact++ netwerk now trained with my own custom dataset, but here is where I get stuck...
To calculate a 3D pose I need to know where the overlayed mask is on the screen. 
I know the segmentation is drawn real time, but is it possible to pass a list of detected objects through to a
c# script for example?

If yes, how and where in the code do I change this without breaking the whole software. I am pretty new to python and
do not want to break the network. I just need the data from the detection and then I can run another neural network for prioritizing which object I want to focus on to calculate the 3D pose for.

So the idea is ==> detection ==> priotise detection through tensorflow(example) ==> openCV 3D calculation

Would love to hear back from some one that knows how to do this.

**Comment by piseabhijeet** [comment]
Hi,

Here is what you can do - get the mask images from YOLACT and project it to your 3D data using the camera intrinsics and extrinsics. So you will get the ROI in 3D - which of course will give occluded points as z (depth) is not known in 2D. You will have to refine the object in 3D space.
